CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## v0.0.16 (12/08/2019)
```

Comment:
I haven’t shipped 0.0.15 yet, so can you move this to the 0.0.15 section?

Reply:
Done.